package.json

```diff
@@ -0,0 +1,62 @@
+{
+  "name": "imagekit-next",
```

Comment:
In other packages react, vue etc we are using convention of `imagekitio-react`, `imagekitio-vue` etc

Reply:
Done, updated it to `imagekitio-next`